src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -158,12 +158,19 @@ const natureOfControlTypeMap = new Map<string, NatureOfControlType>([
   [natureOfControl.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM, NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL]
 ]);
 
-export const mapBoPrivateAddress = (boPrivateData: BeneficialOwnerPrivateData[], ch_reference: string, hasOffice: boolean) => {
-  for (const private_bo_data of boPrivateData) {
-    if (private_bo_data.hashedId === ch_reference) {
-      return hasOffice
-        ? mapBOMOAddress(private_bo_data.principalAddress)
-        : mapBOMOAddress(private_bo_data.usualResidentialAddress);
+export const mapIndividualBOPrivateData = (boPrivateData: BeneficialOwnerPrivateData[], beneficialOwner: BeneficialOwnerIndividual) => {
+  for (const boData of boPrivateData) {
+    if (boData.hashedId === beneficialOwner.ch_reference) {
+      beneficialOwner.usual_residential_address = mapBOMOAddress(boData.usualResidentialAddress);
+      beneficialOwner.date_of_birth = mapInputDate(boData.dateOfBirth);
```

Comment:
Are there any views to update? We may have been setting the DOB day to empty on the check your answers screen

Reply:
> Are there any views to update? We may have been setting the DOB day to empty on the check your answers screen

views/update/review-beneficial-owner-individual.html uses the same file I changed for MOs: views/includes/date_of_birth-display.html

Reply:
> I think AC 3 mentions that in **general**, if no DOB is retrieved then we display month and year @n-r-b @bmcgreevyCH

I spoke to Andre about this and D.O.B should be in CHIPS db. We have no way to check that the day is "missing".